app/pages/nonsipp/shares/WhyDoesSchemeHoldSharesPage.scala

```diff
@@ -45,5 +46,9 @@ case class WhyDoesSchemeHoldSharesPage(srn: Srn, index: Max5000) extends Questio
     }
 
   private def dependantPages(srn: Srn): List[Removable[_]] =
-    List(WhenDidSchemeAcquireSharesPage(srn, index))
+    List(
```

Comment:
Do you need to update WhyDoesSchemeHoldSharesPageSpec after this change ?

Reply:
Hi @cyeniceri I believe we are not updating the this WhyDoesSchemeHoldSharesPageSpec.

Reply:
I think github shows this change with a mistake. I did this change (and updated WhyDoesSchemeHoldSharesPageSpec too) and that change has gone into main already.
@narendrapaduchuri could you please rebase on main? That might remove this change for the PR.